MANIFEST.in

```
@@ -7,6 +7,7 @@ include LICENSE.PSF
 include README.rst
 
 include pyproject.toml
+include py.typed
```

The file is not in the project root, so include doesn't work like that:

``warning: no files found matching 'py.typed'``

```suggestion
include src/cryptography/py.typed
```